ocaml/xapi-cli-server/cli_printer.ml

```diff
@@ -56,7 +56,7 @@ let make_printer sock minimal =
   let multi_line_xapi_minimal pval =
     match pval with
     | PTable rs ->
-        if List.length rs > 0 && List.length (List.hd rs) > 0 then
+        if rs <> [] && List.hd rs <> [] then
```

Comment:
The code below uses List.hd on all the nested lists, they all should be checked:
```suggestion
        if rs <> [] && List.for_all (fun r -> r <> []) rs then
```